### PR TITLE
Beanstalkd driver

### PR DESCRIPTION
**Pull request by Zae**
+ Includes unit test
+ Includes integation test

- Changed some things around to make it easier to test beanstalkd integration (like making fallbackSerializer private)
- includes a integration test that is not run in travis because it does not have a beanstalkd service (use --env beanstalkd in an environment with beanstalkd to do a integration test)
- Does not use the codecept queue module because it uses Pheanstalk 3 and it didn't work correctly.

### CODE REVIEW

**Comment by Zae** [comment]
I don't have a lot of experience with codeception, I have no idea why the tests are failing, locally the are green, except for some filedriver tests, which were broken when I started.

**Comment by initx** [comment]
Hi.
First of all I want to say big thank you for this first PR.

Unfortunately I've made some BC in namespaces before seen this PR and already pushed it to master. 

To fix this I've merged Zae:feature/beanstalkd-driver into initx:feature/beanstalkd-driver and will try to fix all issues which currently occurs.

Will update here with more info after investigation.

**Comment by initx** [comment]
@Zae Please take a look at https://github.com/initx/querabilis/pull/3
It would be great if you could synchronize your Zae:feature/beanstalkd-driver with initx:feature/beanstalkd-driver then I would like to merge this PR.
This will allow us to see your commits in history which I would like to keep.

**Comment by Zae** [comment]
Hi @initx,

I merged your branch into mine.

Did not know it was that easy to just spin up additional docker containers in travis, that's great.
Also adding the beanstalk assertions to the integration tester class is nice, did not know that either.

Hope this helps your package, cheers!

File: tests/_support/InteractsWithBeanstalkd.php [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<?php
```
Comment (initx) [inline tests/_support/InteractsWithBeanstalkd.php]: Hm looks like it is not needed anymore

**Pull request merged**